docs/USAGE.md

```diff
@@ -22,6 +23,8 @@
 
 ⚠️ Note that `--storage_directory` must not be provided if `--url` is not provided.
 
+⚠️ Note that parameters marked with an asterisk (*) in the help menu are mandatory.
```

Comment:
I would add this directly to the `-h` output as well - IMHO the help message should be self-documenting.

Reply:
I could add that to `-h` description. JCommander does not seem to have support to add text between Usage and Options, bu I might be missing something.

Reply:
Could you add as last line?

Reply:
Done :)